ckanext/showcase/templates/showcase/read.html

```diff
@@ -8,6 +8,7 @@
 {% block styles %}
   {{ super() }}
   <link rel="stylesheet" href="{{ g.site_url }}/ckanext_showcase.css" />
+  <link rel="stylesheet" href="{{ g.site_url }}/ckeditor-content-style.css" type="text/css">
```

Comment:
This should only be loaded if editor == 'ckeditor', so you can move the `editor` var setting to the top of the file and use the same check here.